### DISCUSSION TITLE
Show ratings in smart collections

### DISCUSSION THREAD

**Discussion by netplexflix** [meisnate12/Plex-Meta-Manager#588] [Q&A] [answered]
When you edit a smart collection and sort by "audience rating", [all the ratings will show underneath the posters](https://i.ibb.co/27bG0cG/Screenshot-20220108-195934-Chrome.jpg) .
However as soon as you stop editing, and just look at your collection, they disappear.

Is there any setting/string I can use in my PMM yml to show the ratings in the collection?

PS: i tried joining the discord, but the invite is invalid

**Comment by meisnate12** [comment, marked as answer]
i dont think theres anyway to have that happen in plex pmm just emulates what plex already can do
